File: 8주차/MainActivity.java
package com.cookandroid.a1025;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    LinearLayout baseLayout;
    Button button1;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        setTitle("1025 게임프로그래밍");

        baseLayout = (LinearLayout) findViewById(R.id.baseLayout);

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        super.onCreateOptionsMenu(menu);
        MenuInflater mInflater = getMenuInflater();
        mInflater.inflate(R.menu.menu1, menu);
        return true;
    }
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case R.id.itemnum:
                Toast.makeText(getApplicationContext(), "2017748065", Toast.LENGTH_LONG).show();
                return true;
            case R.id.itemname:
                Toast.makeText(getApplicationContext(), "이도건", Toast.LENGTH_LONG).show();

                return true;
            case R.id.iteminfo:
                Toast.makeText(getApplicationContext(), "2017748065 이도건", Toast.LENGTH_LONG).show();
                return true;
        }
        return false;
    }

}

File: project/app/src/main/java/com/cookandroid/project/MainActivity.kt
package com.cookandroid.project

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.Menu
import android.view.MenuItem
import android.view.inputmethod.EditorInfo
import android.webkit.WebViewClient
import kotlinx.android.synthetic.main.activity_main.*
class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        title = "직접검색"

        webView.apply {
            settings.javaScriptEnabled = true
            webViewClient = WebViewClient()
        }

        webView.loadUrl("https://www.naver.com")

        urlEditText.setOnEditorActionListener { _, actionId, _ ->
            if (actionId == EditorInfo.IME_ACTION_SEARCH) {
                webView.loadUrl(urlEditText.text.toString())
                true
            } else {
                false
            }
        }
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.menu, menu)
        return super.onCreateOptionsMenu(menu)
    }
    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            R.id.item1_1 -> {
                val intent = Intent(application, SecondActivity::class.java)
                startActivity(intent)
            }
            R.id.information -> {
                val intent = Intent(application, infoActivity::class.java)
                startActivity(intent)
            }
            R.id.item1_2 ->{
                val intent = Intent(application, ThirdActivity::class.java)
                startActivity(intent)
            }
            R.id.item1_3 ->{
                val intent = Intent(application, FourthActivity::class.java)
                startActivity(intent)
            }
        }
        return super.onOptionsItemSelected(item)
    }
}





